docs/cadence_migration_guide/emulator-state-migration-guide.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Migrating Emulator state to Cadence 1.0
+description: How to locally test storage migration and staged contract upgrades on Emulator
+sidebar_position: 7
```

Comment:
Made this `7`, as https://github.com/onflow/cadence-lang.org/pull/65 will be `6`